nvdaHelper/vbufBackends/mshtml/node.cpp

```diff
@@ -407,13 +384,13 @@ void MshtmlVBufStorage_controlFieldNode_t::preProcessLiveRegion(const MshtmlVBuf
 	} else {
 		this->ariaLiveAtomicNode=parent?parent->ariaLiveAtomicNode:NULL;
 	}
-	LOG_DEBUG(L"preProcessLiveRegion: ariaLiveNode "<<ariaLiveNode<<L", ariaLiveIsTextRelevant "<<ariaLiveIsTextRelevant<<L", ariaLiveIsAdditionsRelevant "<<ariaLiveIsAdditionsRelevant<<L", ariaLiveIsBusy "<<ariaLiveIsBusy<<L", ariaLiveAtomicNode "<<ariaLiveAtomicNode);

```

Comment:
I assume that this line should be restored and the (new) comment should be removed